setup.py

```diff
@@ -301,6 +301,10 @@ def get_extensions():
         if 'nvcc' in extra_compile_args and platform.system() != 'Windows':
             extra_compile_args['nvcc'] += ['-std=c++14']
 
+        # Add -O3 to speed up the runtime
+        extra_compile_args['cxx'] += ['O3']
+        extra_compile_args['nvcc'] += ['-O3']
```

Comment:
Add nvcc flag only when nvcc is available.

Reply:
Thanks for your review. I have fixed that with: 
```python
# Add -O3 to speed up the runtime
    extra_compile_args['cxx'] += ['O3']
if 'nvcc' in extra_compile_args:
    extra_compile_args['nvcc'] += ['-O3']
```